data/src/commonMain/kotlin/com/wire/kalium/logic/data/web/WebEventContent.kt

```diff
@@ -53,63 +53,71 @@ data class WebConversationContent(
 )
 
 @Serializable
-sealed class WebEventContent {
+sealed interface WebEventContent {
 
     @Serializable
-    sealed class Conversation : WebEventContent() {
+    sealed interface Conversation : WebEventContent {
+        val qualifiedConversation: ConversationId
+        val conversation: String
+        val qualifiedFrom: UserId?
 
         @Serializable
         @SerialName("conversation.group-creation")
         data class NewGroup(
-            @SerialName("qualified_conversation") val qualifiedConversation: ConversationId,
-            @SerialName("qualified_from") val qualifiedFrom: UserId?,
+            @SerialName("qualified_conversation") override val qualifiedConversation: ConversationId,
+            @SerialName("conversation") override val conversation: String,
+            @SerialName("qualified_from") override val qualifiedFrom: UserId?,
             @SerialName("from") val from: String,
             @SerialName("data") val members: WebGroupMembers,
             @SerialName("time") val time: String
-        ) : Conversation()
+        ) : Conversation
 
         @Serializable
         @SerialName("conversation.message-add")
         data class TextMessage(
-            @SerialName("qualified_conversation") val qualifiedConversation: ConversationId,
-            @SerialName("qualified_from") val qualifiedFrom: UserId?,
+            @SerialName("qualified_conversation") override val qualifiedConversation: ConversationId,
+            @SerialName("conversation") override val conversation: String,
+            @SerialName("qualified_from") override val qualifiedFrom: UserId?,
             @SerialName("from") val from: String,
             @SerialName("from_client_id") val fromClientId: String?,
             @SerialName("time") val time: String,
             @SerialName("id") val id: String,
             @SerialName("data") val data: WebTextData,
-            @SerialName("reactions") val reactions: Map<String, String>?
-        ) : Conversation()
+            @SerialName("reactions") val reactions: Map<String, String>?,
+            @SerialName("category") val category: Int? // 16 ?
```

Comment:
where did this come from? 🤔

Reply:
Oooh.

All of the `WebEventContent` shouldn't be in this PR.

That came by mistake when taking stuff from the Hackaton :D

I'll remove these.

Reply:
Done!